src/components/ProvisionSmartWalletDialog.tsx

```diff
@@ -0,0 +1,115 @@
+import { Dialog, Transition } from '@headlessui/react';
+import clsx from 'clsx';
+import { Fragment } from 'react';
+import { chainConnectionAtom, provisionToastIdAtom } from 'store/app';
+import { useAtomValue, useSetAtom } from 'jotai';
+import { provisionSmartWallet } from 'services/wallet';
+
+// Increment every time the current terms change.
+export const currentTermsIndex = 1;
+
+export type SwingsetParams = { foo: string };
```

Comment:
Foo?

Reply:
Ah yea, I was originally trying to pass in the whole params, and created this type before I knew how they were structured. I changed this to only pass in the amount value, and handle any parsing errors in `ChainConnection.tsx` where the params are loaded, so this is unused. Removed.